src/containers/MyNdla/Learningpath/components/ResourcePicker.tsx

```diff
@@ -196,6 +196,10 @@ export const ResourcePicker = ({ setResource }: Props) => {
                 if (e.key === "Enter" && !highlightedValue) {
                   onSearch();
                 }
+                if (e.key === "Enter" && highlightedValue) {
+                  const resource = searchHits.find((item) => item.id === highlightedValue) as Resource;
+                  onResourceSelect(resource);
+                }
```

Comment:
Kunne slått disse sammen for sjekk på enter, og så hatt forskjellig oppførsel inni.